src/eastasianwidthdef.c

```diff
@@ -1,6 +1,4 @@
 /*
- * Definitions of internal data types for Indic Conjunct Break.
- *
```

Comment:
Thanks for fixing the comment. But we had better replace the bad comment with the right one.

Reply:
The file now only includes other files so I don't see an urgent need in commenting it.